### PR TITLE
feat: add arbitrum to adapters

### PR DESCRIPTION
**Pull request by doomsower**
**NOTE**

#### Please enable "Allow edits by maintainers" while putting up the PR.

---

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
3. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
4. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
5. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
6. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/gearbox exports TVL: 
        
 
 
    ethereum                  364.31 M
    arbitrum                  2.47 M
    
    total                    366.78 M

File: projects/gearbox/index.js [hunk 1]
```diff
@@ -406,13 +409,17 @@ async function getV3CAs(dc300, creditManager, block, api) {
 }
 
 // src/adapter/index.ts
-async function tvl(api) {
-  const block = api.block
+async function tvl(timestamp, block, _, { api }) {
```
Comment (g1nt0ki) [inline projects/gearbox/index.js:412]: this is mostly good except for this line, the block there is always ethereum block (it is legacy)

so, what you want is `cont block = await api.getBlock()`

**Comment by llamatester** [comment]
The adapter at projects/gearbox exports TVL: 
        
 
 
    ethereum                  362.02 M
    arbitrum                  2.42 M
    
    total                    364.43 M

**Comment by g1nt0ki** [comment]
@doomsower thanks for the PR

**Pull request merged**